templates/zombienet/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "template-zombienet-tests"
+description = "Zombienet test for templates."
+version = "0.0.0"
+license = "Unlicense"
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+edition.workspace = true
+publish = false
+
+[dependencies]
+env_logger = "0.11.2"
+log = "0.4"
+tokio = { version = "1.36.0", features = ["rt-multi-thread"] }
+anyhow = "1.0.81"
+zombienet-sdk = "0.2.6"
+
+[features]
+zombienet = []
```

Comment:
Why does this have a separate feature?

Reply:
The idea is to hide this _integration test_ behind a feature, since require some extra artifacts and steps and we don't want to mess with people running `cargo test`.

Reply:
But if you run `cargo test` in a template, for example `./templates/parachain`, this crate is not part of your working tree and is not tested anyways??